modules/nf-core/glimpse/sample/tests/main.nf.test

```diff
@@ -0,0 +1,63 @@
+nextflow_process {
+
+    name "Test Process GLIMPSE_SAMPLE"
+    script "modules/nf-core/glimpse/sample/main.nf"
+    process "GLIMPSE_SAMPLE"
+    tag "glimpse"
+    tag "glimpse/sample"
+    tag "modules_nfcore"
+
+    options "--sample"
```

Comment:
are you providing`--sample` here or in the nextflow.config ?

Reply:
From the nextflow config sorry.
I tried this but it doesn't work (i forgot to delete it).